Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -12051,6 +12053,104 @@ def get_server_config():
     server_config = body.get('sysConf', {})
     return server_config
 
+
+def mask_secrets(args: tuple, kwargs: dict, func: Callable, secret_keys: Optional[List[str]] = None) -> tuple:
```

Comment:
You can use default for `secret_keys`:
- sensitive_params, in IntegrationLogger.__init__
- keywords_to_censor, in censor_request_logs